### PR TITLE
updated documenation

### PR DESCRIPTION
**Pull request by peuter**
refs #1717

Signed-off-by: Tobias Bräutigam <tbraeutigam@gmail.com>

### CODE REVIEW

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 1]
```diff
@@ -11,38 +11,38 @@ This adds a backend for the web based visualization CometVisu (http://www.cometv
...
 
-* Copy the addon org.openhab.ui.cometvisu*.jar to the openHAB addon folder
-* Then just extract the "release" folder (the one which contains the index.html file) of the downloaded CometVisu archive in the runtime folder off your openHAB2 server. 
-Note: if you want to store the CometVisu client in another folder you have to set the webfolder in your config file
+* Install the CometVisu addon (optionally you can also installe the CometVisu-PHP addon, but it is not required to run the CometVisu)
+* Download the CometVisu release (https://github.com/CometVisu/CometVisu/releases) and extract the "release" folder (the one which contains the index.html file) of the downloaded archive somewhere on your openHAB2 server (make sure that openHAB can access that folder). 
+Note: You have to set the path to this folder in your config file (webFolder).
 
 ## Configuration
 ### Server configuration
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: please add an empty line

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 2]
```diff
@@ -11,38 +11,38 @@ This adds a backend for the web based visualization CometVisu (http://www.cometv
...
-You can customize some setting of the CometVisu backend in a configuration file named 'cometvisu.cfg' which has to be created in the 'conf/services/' folder of your openHAB2 instance.
+You can customize some setting of the CometVisu backend in a configuration file named 'cometvisu.cfg' which has to be created in the 'conf/services/' folder of your openHAB2 instance, if it does not exist.
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: please add an empty line

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 3]
```diff
@@ -194,6 +194,9 @@ Not all of the PHP-based functions in the CometVisu client have been tested so f
 
 The sitemap support can only be used as a starting point for own customizations, e.g. you open an automatic generated config and store it in by CometVisu by using the editor. Then you start to customize it to your own needs. New items must by added by hand from the moment you stored and changed the generated config.
 
+### 404 error
+Please make sure that openhab has the rights to read the configured \<webFolder\> folder.
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: please add an empty line

**Comment by peuter** [comment]
I guess you meant new lines after the headlines, right. I added all missing new lines

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 4]
```diff
@@ -11,38 +11,40 @@ This adds a backend for the web based visualization CometVisu (http://www.cometv
...
 ## Installation
 
-* Copy the addon org.openhab.ui.cometvisu*.jar to the openHAB addon folder
-* Then just extract the "release" folder (the one which contains the index.html file) of the downloaded CometVisu archive in the runtime folder off your openHAB2 server. 
-Note: if you want to store the CometVisu client in another folder you have to set the webfolder in your config file
+* Install the CometVisu addon (optionally you can also installe the CometVisu-PHP addon, but it is not required to run the CometVisu)
+* Download the CometVisu release (https://github.com/CometVisu/CometVisu/releases) and extract the "release" folder (the one which contains the index.html file) of the downloaded archive somewhere on your openHAB2 server (make sure that openHAB can access that folder). 
+Note: You have to set the path to this folder in your config file (webFolder).
 
 ## Configuration
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: one more behind here

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 5]
```diff
@@ -88,6 +93,7 @@ can be shown in the visu
 
 ## Examples
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: one more behind here

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 6]
```diff
@@ -155,6 +163,7 @@ var tag = "Call".encode("UTF-8")
 sendHttpGetRequest("http://<openhab-server>:8080/<webAlias>/plugins/rsslog_pdo.php?c="+content+"&t="+tag)
 ```
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: one more behind here

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 7]
```diff
@@ -187,22 +196,30 @@ sendCommand(Logger,"Received call from 123456789")
 Please note: You have to add the colorchooser plugin in the meta>plugins section of you config
 
 ## Known problems
+
 Not all of the PHP-based functions in the CometVisu client have been tested so far. The untested features are:
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: one more behind here

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 8]
```diff
@@ -187,22 +196,30 @@ sendCommand(Logger,"Received call from 123456789")
...
+
+Please make sure that openhab has the rights to read the configured \<webFolder\> folder.
+
 ### 403 error 
+
 If you get an 403 - Access Denied error, when you try to open the CometVisu in your browser you have not copied the correct release folder into the \<webFolder\> folder. Please check if there is a subfolder with the exact name "release/", which contains an index.html file and copy the content to the folder defined in the \<webFolder\>-property.
 
 ### Hints for development
+
 For every change in the CometVisu XSD-schema the JAXB auto-generation job must be executed:
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: one more behind here

File: addons/ui/org.openhab.ui.cometvisu/README.md [hunk 9]
```diff
@@ -187,22 +196,30 @@ sendCommand(Logger,"Received call from 123456789")
...
 Not all of the PHP-based functions in the CometVisu client have been tested so far. The untested features are:
 * Automatic configuration upgrade, when the library version has changed
 * calendarlist plugin
 * upnpcontroller plugin
 
 The sitemap support can only be used as a starting point for own customizations, e.g. you open an automatic generated config and store it in by CometVisu by using the editor. Then you start to customize it to your own needs. New items must by added by hand from the moment you stored and changed the generated config.
 
+### 404 error
+
+Please make sure that openhab has the rights to read the configured \<webFolder\> folder.
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/README.md]: -> openHAB

**Comment by peuter** [comment]
Ok, I hope I got them all now.

**Pull request merged**